### ISSUE TITLE
CocoaPods installation fails

### ISSUE THREAD

**Issue by mitchellporter** [IBAnimatable/IBAnimatable#89]
I get the following error when trying to install via CocoaPods:

```
[!] Unable to satisfy the following requirements:

- `IBAnimatable (~> 1.0)` required by `Podfile`

Specs satisfying the `IBAnimatable (~> 1.0)` dependency were found, but they required a higher minimum deployment target.
```

My current deployment target is iOS 8.0

**Comment by tbaranes** [comment]
The last released of `IBAnimatable` on CocoaPods was targeting iOS 9.0. However, the next one will be compatible with iOS8.0. 

Until this new release, you can still use it with CocoaPods by targeting the master directly:

```
pod 'IBAnimatable', :git => 'https://github.com/JakeLin/IBAnimatable'
```

**Comment by JakeLin** [comment]
@mitchellporter thanks for reporting this issue, we have released v1.1 which supports iOS 8. And we also pushed the podspec. You should be able to `pod 'IBAnimatable'` now. please let me know if you have any question.

@tbaranes thanks for troubleshooting this issue.

**Comment by mitchellporter** [comment]
@JakeLin @tbaranes thanks for the help!
